client/pom.xml

```diff
@@ -33,6 +33,17 @@
     </properties>
 
     <dependencies>
+        <dependency>
+            <groupId>com.google.guava</groupId>
+            <artifactId>guava</artifactId>
+            <version>23.0</version>
```

Comment:
If I remember correctly, guava has been relied on. No need to do it.

Reply:
done